src/test/kotlin/no/nav/faktureringskomponenten/service/mappers/FakturalinjeMapperTest.kt

```diff
@@ -40,4 +41,30 @@ class FakturalinjeMapperTest {
                 beskrivelse.shouldBe("Inntekt: 80000, Dekning: HELSE_OG_PENSJONSDEL, Sats: 28.3 %")
             }
     }
+
+    @Test
+    fun `to perioder med samme fom og tom datoer`() {
+        val fra = LocalDate.of(2023, 1, 1)
+        val til = LocalDate.of(2023, 1, 31)
+        val perioder = listOf(
+            FakturaseriePeriode(
+                enhetsprisPerManed = BigDecimal(22830),
+                startDato = fra,
+                sluttDato = til,
+                beskrivelse = "Inntekt: 80000, Dekning: Pensjonsdel, Sats: 21.5 %"
+            ),
+            FakturaseriePeriode(
+                enhetsprisPerManed = BigDecimal(25470),
+                startDato = fra,
+                sluttDato = til,
+                beskrivelse = "Inntekt: 80000, Dekning: Helse- og pensjonsdel, Sats: 28.3 %"
+            )
+        )
+
+        val fakturaLinjer = FakturalinjeMapper().tilFakturaLinjer(perioder, fra, til)
```

Comment:
Ville kanskje ikke bruke samme `fra` og `til` i kall til FakturaLinjer siden det som trengs er at at `fra` og `til` er innenfor perioden man lager faktura for